src/strings/PowerFxResources.en-US.resx

```diff
@@ -5004,6 +5004,22 @@
     <value>Contrast theme</value>
     <comment>Display text representing the Contrast value of TeamsTheme enum (TeamsTheme_Contrast_Name). The possible values for this enumeration are: Default, Dark, Contrast.</comment>
   </data>
+  <data name="AboutPatch" xml:space="preserve">
+    <value>Applies the updates to the specified row and changes the row in the specified datasource. Returns the updated row.</value>
+    <comment>Description of 'Patch' function.</comment>
+  </data>
+  <data name="PatchDataSourceArg" xml:space="preserve">
+    <value>Data source</value>
+    <comment>Required. The data source that contains the records that you want to modify or will contain the records that you want to create.</comment>
```

Comment:
>Required. The data source that contains the records that you want to modify or will contain the records that you want to create. [](http://example.com/codeflow?start=13&length=128)

The comment field is not something that is shown to the users; instead, it is context that we give to the translators so that they are able to translate the <value> correctly.